packages/@n8n/config/src/configs/workflow-history.config.ts

```diff
@@ -0,0 +1,12 @@
+import { Config, Env } from '../decorators';
+
+@Config
+export class WorkflowHistoryConfig {
```

Comment:
we don't need to make this change now, but workflow-history should become a module at some point, and when it does, it might make sense to move this file into `packages/cli/src/workflows/workflow-history.ee/`.

Reply:
Exactly, as we move towards modules, the respective configs will live there